config.schema.json

```
@@ -128,20 +128,41 @@
 				"default": false,
 				"required": false
 			},
-			"enableClimateReactSwitch": {
-				"title": "Enable Climate React Switch",
-				"description": "Add Climate React (Smart mode) switch",
-				"type": "boolean",
-				"default": false,
-				"required": false
-			},
             "enableClimateReactAutoSetup": {
                 "title": "Enable Climate React Auto Setup",
                 "description": "Auto-update the Climate React (Smart mode) configuration whenever the AC state is set or changed",
                 "type": "boolean",
                 "default": false,
                 "required": false
             },
+			"climateReactAutoSetupOffset": {
+                "title": "Climate React Auto Setup Offset",
+                "description": "Offset, in degrees, above or below AC target temperature that will be used to toggle AC when `enableClimateReactAutoSetup` is enabled",
+                "type": "number",
```

I expect that this should be a whole number, right?
```suggestion
                "type": "integer",
```

And is a default of 0 correct (also saw that in other places) shouldn't it default to 1 like it is currently? If not, wouldn't that be a non-backwards compatible change?

---

The offset can in theory be a non-integer, e.g. 1.5 degrees. By default you don't have any offset to the auto setup range, so 0 makes sense. When setting it to 0 it has not impact on the bounds calculation and so preserves the prior behavior.